components/VotesList/VotesListItem.tsx

```diff
@@ -389,7 +389,7 @@ export function VotesListItem({
             <TextInput
               value={selectedVote ?? getDecryptedVoteAsString() ?? ""}
               onInput={selectVote}
-              onClear={exitCustomInput}
+              onClear={isCustomInput ? exitCustomInput : undefined}
```

Comment:
Only provide onClear when we are dealing with a custom input.

Reply:
in the story book link its showing an x on some inputs without a dropdown 
![image](https://user-images.githubusercontent.com/4429761/233369093-be404dce-4c58-42d5-8d9e-2920e0116f09.png)